src/HomePage.js

```diff
@@ -113,16 +134,17 @@ export class HomePage extends React.Component {
                 
                 {idCard && 
                 <ModalCard 
-                    onCloseModal={this.handleCloseModal} 
-                    onChangeCard={this.handleChangeCard}
-                    {...findCardById(idCard, dataByStatuses)}
+                    onCloseModal={this.handleCloseModal}                    
+                    forChildren={
+                        {
+                            changeDescription: true,
+                            statuses,
+                            card: {...findCardById(idCard, dataByStatuses)},
+                            onChangeCard: this.handleChangeCard
+                        }
+                    }                    
```

Comment:
it doesn't make sense to have this prop since you pass directly props to children (you can do this here, no need to do it in ModalCard)